ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -92,6 +92,25 @@ OnboardingBasePage {
             }
         }
 
+        StatusBaseText {
+            id: errorTxt
+            anchors.horizontalCenter: parent.horizontalCenter
+            font.pixelSize: 12
+            color: Theme.palette.dangerColor1
+            onTextChanged: {
+                if(text === "") filler.visible = true
+                else filler.visible = false
+            }
+        }
+
+        // Just a column filler to keep the component height althought errorTxt.text is ""
+        Item {
+            id: filler
+            width: root.width
+            visible: true
```

Comment:
Can this be changed to `visible: errorTxt.text === ""`? Then we could remove the `onTextChanged`